CHANGELOG.md

```diff
@@ -2512,9 +2512,11 @@ Swift 2.2
     AnyObject>`.  It bridges in O(1), without memory allocation.
 
   - `Dictionary<K, V>` has an implicit conversion to `NSDictionary`.
-    `Dictionary<K, V>` bridges to `NSDictionary` iff both `K` and `V` are
+    `Dictionary<K, V>` bridges to `NSDictionary` [iff] both `K` and `V` are
```

Comment:
Thanks for feedback.
I think that this also works. Please see,
https://github.com/takecian/swift/blob/master/CHANGELOG.md